lib/phoenix_live_view/plug.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Phoenix.LiveView.Plug do
+  @moduledoc """
```

Comment:
My suggestion is to keep it `@moduledoc false` for now until we have a use case to expose it. :)